app/views/subscription_mailer/send_digest.html.erb

```diff
@@ -1,10 +1,12 @@
 <header style="text-align: center;color:black">
   <img src="https://avatars2.githubusercontent.com/u/4621650?s=200&v=4" style="width: 100px;height: 100px">
   <p style="font-size: 20px">[Public Lab] <%= @subject %></p>
+  <p><%= Time.now.strftime("%B %-d %Y") %></p>
 </header>
 
 <div style="width:95%;margin: 0 auto;margin-top: 54px;color: black;">
   <p style="font-size: 1.2em; font-family: sans-serif;margin-bottom: 0;color: lightgray;">Top picks for you</p>
```

Comment:
```suggestion
```

Hey @stoic-plus, you are supposed to remove this line as per issue- https://github.com/publiclab/plots2/issues/5340.